t/t5701-git-serve.sh

```diff
@@ -8,13 +8,23 @@ export GIT_TEST_DEFAULT_INITIAL_BRANCH_NAME
 . ./test-lib.sh
 
 test_expect_success 'test capability advertisement' '
+	printf "agent=git/$(git version | cut -d" " -f3)" >agent_os_name &&
+	if test_have_prereq WINDOWS
+	then
+		test_config transfer.advertiseOSVersion false
```

Comment:
Here also I think we should use `git config` instead of `test_config` to disabling transfer.advertiseOSVersion on Windows, for the same reason as in my comment above.

Reply:
And yeah, here also it might be worth telling in a comment that we use `git config` instead of `test_config` so that other tests below can reuse the 'expect.base' file.

Reply:
Yeah, but, will git config be enough to test the transfer.advertiseOSVersion or we will need to test in another way ?

Reply:
Yeah, I think we might need to add other tests to test `transfer.advertiseOSVersion` but we can do that later. I think it will become clearer how we should test everything when we have `osversion.command` and `osversion.format`.

Reply:
Ok, noted, thank you.